app/views/download/package.erb

```diff
@@ -122,7 +122,7 @@ pacman -S <%= repo_name %>/<%= @package %></pre>
             when 'CentOS', 'RHEL', 'SL'
               "cd /etc/yum.repos.d/\nwget #{v[:repo]}#{@project}.repo\nyum install #{@package}"
             when 'Debian', 'Univention'
-              "echo 'deb #{v[:repo].gsub(/(\w):(\w)/, '\1:/\2').gsub(/^https/, 'http')} /' > /etc/apt/sources.list.d/#{@package}.list \napt-get update\napt-get install #{@package}"
+              "echo 'deb #{v[:repo].gsub(/(\w):(\w)/, '\1:/\2').gsub(/^https/, 'http')} /' > /etc/apt/sources.list.d/#{@project}.list\napt-get update\napt-get install #{@package}"
```

Comment:
In this line I also removed a space that's not present in the equivalent line in the Ubuntu instructions.